homeassistant/util/yaml.py

```diff
@@ -6,6 +6,7 @@
 import glob
 import yaml
 
+import homeassistant.util.secrets as secrets
```

Comment:
This file is a YAML util, not a Home Assistant util. I think that we can reduce it to a single function as described in my other comment. I hope we then don't need this file anymore.